server/asset/dcr/script.go

```diff
@@ -97,18 +121,14 @@ type dcrScriptAddrs struct {
 // Extract the addresses from the pubkey script, or the redeem script if the
 // pubkey script is P2SH. Addresses can be of several types, but the types
 // suppported will be pubkey
-func extractScriptAddrs(scriptType dcrScriptType, pkScript, redeemScript []byte) (*dcrScriptAddrs, error) {
+func extractScriptAddrs(script []byte) (*dcrScriptAddrs, error) {
 	var err error
 	var addrs []dcrutil.Address
 	var numRequired int
 	pubkeys := make([]dcrutil.Address, 0)
 	pkHashes := make([]dcrutil.Address, 0)
 	// For P2SH and non-P2SH multi-sig, pull the addresses from the pubkey script.
-	if scriptType.isP2PKH() {
-		_, addrs, numRequired, err = txscript.ExtractPkScriptAddrs(0, pkScript, chainParams)
-	} else {
-		_, addrs, numRequired, err = txscript.ExtractPkScriptAddrs(0, redeemScript, chainParams)
-	}
+	_, addrs, numRequired, err = txscript.ExtractPkScriptAddrs(0, script, chainParams)
```

Comment:
Might as well eliminate the `var` declarations above.

Reply:
whoops